css/style.css

```diff
@@ -0,0 +1,2921 @@
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Montserrat Regular'), local('Montserrat-Regular'),
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2'), 
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff') format('woff'); 
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)